internal/pipe/ko/ko.go

```diff
@@ -493,3 +511,33 @@ func validateMainPath(path string) error {
 	}
 	return nil
 }
+
+func copyImage(src, dst string) (string, error) {
+	log.WithField("src", src).
+		WithField("dst", dst).
+		Info("copying manifest")
+	if err := crane.Copy(src, dst, crane.WithAuthFromKeychain(keychain)); err != nil {
+		return "", fmt.Errorf("ko: could not copy %q to %q: %w", src, dst, err)
+	}
+	digest, err := crane.Digest(dst, crane.WithAuthFromKeychain(keychain))
+	if err != nil {
+		return "", fmt.Errorf("ko: could get digest of %q: %w", dst, err)
```

Comment:
I believe here is missed `not`:
```suggestion
		return "", fmt.Errorf("ko: could not get digest of %q: %w", dst, err)
```